build.sh

```diff
@@ -1,3 +1,6 @@
 #!/usr/bin/env bash
 
-docker build --build-arg TRAVIS_BUILD_NUMBER=$TRAVIS_BUILD_NUMBER --build-arg MYGET_API_KEY=$MYGET_API_KEY .
+docker build \
+  --build-arg TRAVIS_OS_NAME=$TRAVIS_OS_NAME \
+  --build-arg TRAVIS_BUILD_NUMBER=$TRAVIS_BUILD_NUMBER \
+  --build-arg MYGET_API_KEY=$MYGET_API_KEY .
```

Comment:
One problem here is that arguments supplied to `build.sh` are not being passed through. That needs to happen if you want to pass any args to Bullseye, otherwise [you're missing out on all this](https://github.com/adamralph/bullseye/blob/3261c9f4136d932fc6586e1e485305c813972c52/Bullseye/Internal/TargetCollectionExtensions.cs#L171-L197). `build.cmd` has the same problem.